website/docs/index.html.markdown

```diff
@@ -17,15 +17,11 @@ Use the navigation to the left to read about the available resources.
 ## Example Usage
 
 ```hcl
-# Configure the AWS Provider
-provider "aws" {
-  access_key = "${var.aws_access_key}"
-  secret_key = "${var.aws_secret_key}"
-  region     = "us-east-1"
-}
-
 # Create a web server
 resource "aws_instance" "web" {
+  ami           = "ami-2757f631"
```

Comment:
I would discourage hardcoding an AMI ID configuration that can/will become out of date or be potentially incorrect for environments (`ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-20170307` in `us-east-1`). We may want to consider choosing another resource if we are trying to show a more valid configuration such as:

```hcl
resource "aws_vpc" "example" {
  cidr_block = "10.0.0.0/16"
}
```

Reply:
Fixed in c89ce9076